.github/ISSUE_TEMPLATE/0_use_the_issue_template.md

```diff
@@ -0,0 +1,15 @@
+---
+name: CHOOSE ONE OF THE ISSUE TEMPLATES
+about: DON'T IGNORE OR REMOVE THE TEMPLATE, OR RISK YOUR ISSUE BEING REJECTED
```

Comment:
I think we should be more forceful here:
`DON'T IGNORE OR REMOVE THE TEMPLATE, EXCLUDING TEMPLATE DATA WILL RESULT IN THE ISSUE BEING CLOSED`